CLI can not be started when installed via package

```
/usr/local/share/.config/yarn/global/node_modules/ts-node/src/index.ts:261
    return new TSError(diagnosticText, diagnosticCodes)
           ^
TSError: ⨯ Unable to compile TypeScript:
error TS5058: The specified path does not exist: '/usr/local/share/.config/yarn/global/node_modules/@iov/cli/tsconfig_repl.json'.

    at createTSError (/usr/local/share/.config/yarn/global/node_modules/ts-node/src/index.ts:261:12)
    at Object.register (/usr/local/share/.config/yarn/global/node_modules/ts-node/src/index.ts:265:42)
    at new TsRepl (/usr/local/share/.config/yarn/global/node_modules/@iov/cli/src/tsrepl.ts:31:30)
    at Object.main (/usr/local/share/.config/yarn/global/node_modules/@iov/cli/src/cli.ts:202:3)
    at Object.<anonymous> (/usr/local/share/.config/yarn/global/node_modules/@iov/cli/bin/iov-cli:6:5)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
```

--- a/packages/iov-cli/package.json
+++ b/packages/iov-cli/package.json
@@ -27,6 +27,7 @@
   "files": [
     "build/",
     "types/",
+    "tsconfig_repl.json",
     "*.md",
     "!*.spec.*",
     "!**/testdata/"

